archive-dssspell.sh

```diff
@@ -4,6 +4,7 @@ set -e
 if [[ -z "$1" ]]; then
   echo "You must provide a date (YYYY-MM-DD) option to name the directory"
 else
+  date="$1"
```

Comment:
I think we don't need to add this as it can be done by following the specs in the checklist:
`make date="YYYY-MM-DD" archive-spell `
it's a common pattern used also in tests `make test match=`

Reply:
Yeah maybe we should roll it back as what was and then decide if we want to change this in a new PR.